src/components/songs/SongLyric.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function SongLyrics({ songLyrics }) {
+  return (
+    <h1>{songLyrics}</h1> 
```

Comment:
```suggestion
    <pre>{songLyrics}</pre> 
```
A ```pre``` or ```p``` tag may be more appropriate here. Typically, you don't want to use a heading tag to style text content for accessibility/semantic reasons.